acceptance/openstack/sharedfilesystems/v2/shareaccessrules.go

```diff
@@ -16,3 +19,55 @@ func ShareAccessRuleGet(t *testing.T, client *gophercloud.ServiceClient, accessI
 
 	return accessRule, nil
 }
+
+// AccessRightToShareAccess is a helper function that converts
+// shares.AccessRight into shareaccessrules.ShareAccess struct.
+func AccessRightToShareAccess(accessRight *shares.AccessRight) *shareaccessrules.ShareAccess {
+	return &shareaccessrules.ShareAccess{
+		ShareID:     accessRight.ShareID,
+		AccessType:  accessRight.AccessType,
+		AccessTo:    accessRight.AccessTo,
+		AccessKey:   accessRight.AccessKey,
+		AccessLevel: accessRight.AccessLevel,
+		State:       accessRight.State,
+		ID:          accessRight.ID,
+	}
+}
```

Comment:
This is a bit confusing -- we have `shares.AccessRight` and `shareaccessrules.ShareAccess`. `shareaccessrules` describes a new API, so technically it ought to be its own package and struct, but in the end it describes the same information as `shares.AccessRight` does. I'm just pointing this out, but if reviewers are ok with this split, we can move on.

Reply:
Fine by me. We can always split that into a separate package later if that bothers us.